model-optimizer/mo/utils/ir_reader/extenders/topk_extender.py

```diff
@@ -24,7 +24,7 @@ class TopKExtender(Extender):
 
     @staticmethod
     def extend(op: Node):
-        if op.graph.graph['cmd_params'].framework in ('tf', 'caffe'):
+        if op.graph.graph['cmd_params'].framework in ('tf', 'caffe', 'onnx'):
```

Comment:
This is not correct. What if the original ONNX model has operation "TopK"? In this case we should keep both outputs! The same comment for TF model.

The Extender logic to check whether the output with values should be removed or not based on connection of the outputs.

Reply:
Ok, I changed this. @achetver Can you check this condition? It should be consistent with code in TopKNormalizer.

Reply:
Yes, checked, in works correct for models with TopK operation with both one and two outputs.